doc/RELEASE_NOTE.md

```diff
@@ -0,0 +1,905 @@
+As a consequence, Z/IP clients must now adhere to the requirements 
+in SDS11846: Z-Wave Plus Role Type Specification.
+
+## 5 Using This Release
+
+The Z/IP Gateway Software Developers Kit contains the following components:
+
+- Debian packages:
+  - Z/IP Gateway 7.18.03, for Raspberry Pi 3B+
+  - Z/IP Gateway 7.18.03, for Linux i386
+
+- Source code:
+  - Z/IP Gateway 7.18.03
+
+- libzwaveip 7.18.03
+
+- Documentation:
+  - INS12503, Z/IP Gateway Porting Process
+  - SDS12938, Z/IP LAN Security
+  - SDS12089, Z/IP Gateway Bootstrapping
+  - SDS11756, Z/IP DNS Discovery support (DNS-SD, mDNS)
+  - SDS11633, Z/IP Resource Directory (RD, DNS-SD, mDNS)
+  - SDS11445, IP Architecture Framework for Z-Wave (Z/IP)
+  - SDS13944, Node Provisioning Information Type Registry (QR code, Z/IP Gateway, SmartStart)
+  - Z/IP Security 2 in Z/IP Gateway
+  - Z/IP Gateway Source Code Documentation
+  - Z/IP Gateway User Guide (Z-Wave & IP Basics, Compilation, Installation, Troubleshooting, Sample Code)
+  - libzwaveip documentation in the file libzwaveip-7.18.01.release-docs.zip
+    Additionally, the following documentation can be found in:
+    https://www.silabs.com/products/wireless/mesh-networking/z-wave/specification:
+  - Z-Wave Command Class specifications
+
+### 5.1 Key Features of Z/IP Gateway SDK
+
+The Z/IP Gateway SDK 7.18.03 provides the following features:
+
+- Transparent gateway: IP applications reach Z-Wave nodes via IP addresses.
+  Z/IP Packets originating from LAN IP applications are terminated 
+  and forwarded as Z-Wave commands to Z-Wave Nodes.
+  See the Z/IP Command Class Specification.
+  - Supports only Z-Wave Single cast
+  - Z-Wave nodes are identified by IPv6 and IPv4 host addresses
+  - IPv6 and IPv4 Ping (ICMP Echo) support
+  - Z/IP-ND: IPv6 and IPv4 address resolution for Z/IP Applications
+- SmartStart Auto-inclusion; Supports only Add Node
+- SIS support
+- Security 0 and Security 2 support
+- Remote Access and Configuration
+- Forwarding of unsolicited messages to up to two IP destinations
+- Resource & Service Discovery (mDNS) (see SDS11756: Silicon Labs, Z/IP DNS Discovery support (DNS-SD, mDNS))
+- IP Association Proxy
+- Mail Box Service – Support for Non-Listening nodes
+- Z/IP LAN Security using DTLS
+- Installation and Maintenance framework
+- NVM Backup
+- Firmware Update of Z-Wave module attached to the Z/IP Gateway
+- Historical and current transmission statistics
+- Wi-Fi support for connecting ZIP Clients to ZIP Gateway.
+- Migration to Z/IP Gateway 7.11.x from non-Z/IP Gateways (See section 5.1.5).
+- Back-up/Recover (See section 5.1.6)
+- Z-Wave Long Range support
+
+The Z/IP Gateway implements support for two sets of command classes, one set for the IP / LAN side and one set for the Z-Wave /
+PAN side.
+
+#### 5.1.1 LAN-Side Features
+
+The command classes listed in Table 1 is supported on the LAN side of the Z/IP Gateway.
+
+Table 1. Supported Command Classes on LAN Side
```

Comment:
Table 1 in section 5.1.1 is not properly rendered.

Reply:
It is:

https://github.com/rzr/zipgateway/blob/review/doc/rn/main/doc/RELEASE_NOTE.md

but not in devel/main (I just need to rebase it)